Makefile

```
@@ -69,12 +74,12 @@ endif
 endif
 	$(GCOV) --version
 
-qa: lint black test doc
+qa: doc #lint check-format test doc
```

Is it intended to comment out QA tasks other than doc?

```suggestion
qa: doc lint check-format test doc
```